File: IT/Stringhe/Esercizi/C++/esercizio6.cpp
/*
file esercizio6.cpp
compilare con: g++ -std=c++11 esercizio6.cpp
*/

#include <iostream>

using std::string;
using std::cout;
using std::endl;
using std::stoi;

int main()
{
	string s = "12345";

	int n = stoi(s);

	if (n % 2 == 0) {
		cout << "Il numero " << s << " è pari." << endl;
	}
	else {
		cout << "Il numero " << s << " è dispari." << endl;
	}
}

File: IT/Stringhe/Esercizi/Java/Esercizio6.java
/* file Esercizio6.java */

public class Esercizio6 {
	public static void main(String [] args) {
		String s = "12345";
		int n = Integer.parseInt(s);

		if (n % 2 == 0) {
			System.out.println("Il numero " + s + " è pari.");
		}
		else {
			System.out.println("Il numero " + s + " è dispari.");
		}
	}
}

